test/blackbox-tests/test-cases/melange/enabled_if.t

```diff
@@ -34,7 +34,13 @@
   > EOF
 
   $ dune build @melange --display short
+  Error: Alias "melange" specified on the command line is empty.
+  It is not defined in . or any of its descendants.
+  [1]
```

Comment:
These errors might be more meaningful to users than the previous behavior (not building anything because the alias is "empty", but not erroring either).